util/converts.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2018 The ontology Authors
+ * This file is part of The ontology library.
+ *
+ * The ontology is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
```

Comment:
this would make your fork of `wagon` only releasable under the LGPL3 (and not under Apache)

just saying...